scalafix/rules/src/main/scala/fix/Stable212Base.scala

```diff
@@ -45,17 +45,21 @@ trait Stable212Base extends CrossCompatibility { self: SemanticRule =>
   val traversable = exact(
     "_root_.scala.package.Traversable#",
     "_root_.scala.collection.Traversable#",
-    "_root_.scala.package.Iterable#",
-    "_root_.scala.collection.Iterable#"
```

Comment:
Why do we remove these?

Reply:
we use this to detect if we need to add the compat import for 

```scala
     ctx.replaceSymbols(
        "scala.Traversable"            -> "scala.Iterable",
        "scala.collection.Traversable" -> "scala.collection.Iterable"
      )
```

It was incorrectly adding compat import.